doc/source/index.rst

```diff
@@ -283,6 +283,12 @@ Getting Involved
    raysgd/raysgd_tensorflow.rst
    raysgd/raysgd_ref.rst
 
+.. toctree::
+   :maxdepth: -1
+   :caption: Ray Dashboard
+
+   ray-dashboard.rst
```

Comment:
can you move this to a page under Ray Core (specifically right under `configure.rst`)